packages/@react-aria/table/docs/useTable.mdx

```diff
@@ -488,12 +492,826 @@ function Checkbox(props) {
+        overflow: 'hidden',
+        whiteSpace: 'nowrap',
+        textOverflow: 'ellipsis',
+        display: 'block',
+        flex: '0 0 auto',
+        boxSizing: 'border-box'
+        /*- end highlight -*/
+      }}
+      ref={ref}>
+      {cell.rendered}
+    </td>
+  );
+}
+```
+
+</details>
+
+### Resizable checkbox
+
+The `TableCheckboxCell` and the `TableSelectAllCell` now have their widths dictated by the state returned by <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useTableColumnResizeState} />
+and receive minor style changes to accommodate that.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+function ResizableTableCheckboxCell({cell, state, widths}) {
+  let ref = useRef();
+  let {gridCellProps} = useTableCell({node: cell}, state, ref);
+  let {checkboxProps} = useTableSelectionCheckbox({key: cell.parentKey}, state);
+  /*- begin highlight -*/
+  let column = cell.column;
+  /*- end highlight -*/
+
+  return (
+    <td
+      {...gridCellProps}
+      /*- begin highlight -*/
+      style={{
+        width: widths.get(column.key),
+        display: 'flex',
+        boxSizing: 'border-box'
+      }}
+      /*- end highlight -*/
+      ref={ref}>
+
+      <Checkbox
+        {...checkboxProps}
+        /*- begin highlight -*/
+        style={{height: 'fit-content', marginTop: 'auto', marginBottom: 'auto'}}
+        /*- end highlight -*/
+      />
+    </td>
+  );
+}
+```
+
+</details>
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+function ResizableTableSelectAllCell({column, state, widths}) {
+  let ref = useRef();
+  let {columnHeaderProps} = useTableColumnHeader({node: column}, state, ref);
+  let {checkboxProps} = useTableSelectAllCheckbox(state);
+
+  return (
+    <th
+      {...columnHeaderProps}
+      /*- begin highlight -*/
+      style={{
+        width: widths.get(column.key),
+        boxSizing: 'border-box',
+        margin: 'auto'
+      }}
+      /*- end highlight -*/
+      ref={ref}>
+      {state.selectionManager.selectionMode === 'single'
+        ? <VisuallyHidden>{checkboxProps['aria-label']}</VisuallyHidden>
+        /*- begin highlight -*/
+        : <Checkbox {...checkboxProps} style={{height: 'fit-content', marginTop: 'auto', marginBottom: 'auto'}} />
+        /*- end highlight -*/
+      }
+    </th>
+  );
+}
+```
+
+</details>
+
+
+And with that, all necessary changes to the previous table implementation have been made and we now have a table that supports resizable columns!
+The example below supports resizing via mouse, keyboard, touch, and screen reader interactions. Previous behaviors such as selection and sorting are
+also still supported.
+
+```tsx example
+<ResizableColumnsTable aria-label="Table with selection and resizable columns" selectionMode="multiple">
```

Comment:
Hrm, experience w/ the resizer is broken in Safari + iffy in VO and Talkback, digging